src/mono/netcore/sample/Android/Makefile

```diff
@@ -2,10 +2,7 @@ MONO_CONFIG=Release
 MONO_ARCH=x64
 DOTNET := ../../../../.././dotnet.sh
 
-all: runtimepack run
-
-runtimepack:
-	../../../../.././build.sh Mono+Libs -os Android -arch $(MONO_ARCH) -c $(MONO_CONFIG)
+all: run
```

Comment:
This was a helper target used during development.

Reply:
it's building quite a lot of stuff, I thought everyone is using the `make runtime` target in src/mono/wasm/Makefile now :)

if you think it's useful I can bring this back and call that target instead.